docs/UserGuide.md

```diff
@@ -368,121 +341,100 @@ Format:
 ![stat example](images/StatUi.png)
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-If only buyers or only properties are visible, `stat` automatically presents the only buyers/only properties view.
+If only buyers or only properties are visible, `stat` automatically presents the only buyers or only properties view.
 </div>
 
-### Locating properties/buyers by name, tags, price: `find`
+### Locating properties/buyers by name, tags and price: `find`
+
+Use this command to filter or search properties or buyers. This command filters **displayed** properties or buyers whose names contain any of the given keywords, has all the specified tags and whose price is within the specified price range.
```

Comment:
given that the format has max and min price, i think it is ok to stick to price to prevent additional inconsistency.

a budget is just a max price you are willing to pay right? :)

Reply:
will fix actually